tests/unit/network.test.ts

```diff
@@ -0,0 +1,113 @@
+import { getL1Network, addCustomNetwork, getL2Network } from '../../src'
+
+import { expect } from 'chai'
+import { getNetwork } from '../../src/lib/dataEntities/networks'
+
+const mainnetId = 1
+const arbOneId = 42161
+const mockOrbitChainId = 99999999
+
+describe('Network', () => {
+  const fetchErrorMessage =
+    'Network fetched successfully but the chain ID is invalid.'
+
+  it('Adds a custom Orbit chain', async function () {
+    const arbOneNetwork = await getL2Network(arbOneId)
+
+    addCustomNetwork({
+      customL2Network: {
+        // we partially copy Arbitrum One network because we only want to mimic a custom chain
+        ...arbOneNetwork,
+        chainID: mockOrbitChainId,
+        partnerChainID: arbOneId,
+        isArbitrum: true,
+        isCustom: true,
+        isOrbit: true,
+      },
+    })
```

Comment:
shouldn't there be a check for `Networks` containing this network after `addCustomNetwork`?